internal/monitors/appmesh/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+monitors:
+- dimensions: null
+  doc: |
+
+    This monitor starts a StatsD monitor to listen to StatsD metrics emitted
+    by AWS AppMesh Envoy Proxy.
```

Comment:
This is a good place to explain to users how to configure Envoy to send the metrics to the agent.

Reply:
Done. @keitwb